extensions/ComponentWatching/Extension.pm

```diff
@@ -498,6 +501,10 @@ sub _getWatches {
         push @watches, \%watch;
     }
 
+    if ($watch_id) {
+        return @watches->[0] || {};
```

Comment:
Using an array as a reference is deprecated at /app/extensions/ComponentWatching/Extension.pm line 505.

return $watches[0] || {};